proposals/vector_reproject.json

```
@@ -0,0 +1,97 @@
+{
+    "id": "vector_reproject",
+    "summary": "Reprojects the geometry dimension",
+    "description": "Converts the geometries stored in a geometry dimensions to a different projection.\n\nTo spatially align two data cubes with each other, better use the process ``resample_cube_spatial()``.",
```

```suggestion
    "description": "Converts the geometries stored in a geometry dimensions to a different projection.",
```

resample_cube_spatial doesn't support vector data, so the note here doesn't make sense.